acapy_controller/protocols.py

```diff
@@ -394,8 +404,51 @@ async def indy_anoncred_credential_artifacts(
     cred_def_tag: Optional[str] = None,
     support_revocation: bool = False,
     revocation_registry_size: Optional[int] = None,
+    anoncreds_wallet: bool = False,
```

Comment:
I think it would be handy to automatically determine whether we're using an AnonCreds wallet type or not. I believe that information would be available to us by hitting the `GET /status/config` endpoint and checking the value of `response["config"]["wallet.type"]` which will be either `askar` or `askar-anoncreds`.

We could probably also cache this on the controller object, since it's not likely to change. We could add that to the controller setup:

https://github.com/Indicio-tech/acapy-minimal-example/blob/3bfe2895dbebdb61e5d2d3e4f0e89a240595a2e5/acapy_controller/controller.py#L235-L244